Makefile

```diff
@@ -29,6 +29,22 @@ test:
 	docker-compose up -d db queue
 	docker-compose -f docker-compose.test.yml up --build --force-recreate --exit-code-from unit_test unit_test
 
+postman:
```

Comment:
This will need a `docker-compose up -d` added to it, to ensure the app is running before running tests.

Reply:
No, we don't.  If we are running postman against dev having it running locally is completely unnecessary and will have no positive effect.  If it isn't running on dev then local docker commands won't solve that.